misk-redis-testing/src/test/kotlin/misk/redis/AbstractRedisTest.kt

```diff
@@ -1,37 +1,23 @@
 package misk.redis
 
 import com.google.inject.Module
-import com.google.inject.util.Modules
-import misk.testing.MiskTest
-import misk.testing.MiskTestModule
-import misk.time.FakeClockModule
 import okio.ByteString.Companion.encodeUtf8
+import org.assertj.core.api.Assertions
 import org.assertj.core.api.Assertions.assertThat
 import org.assertj.core.api.Assertions.assertThatThrownBy
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.assertThrows
 import redis.clients.jedis.args.ListDirection
-import wisp.time.FakeClock
-import java.time.Duration
-import javax.inject.Inject
-import kotlin.random.Random
 import kotlin.test.assertEquals
 import kotlin.test.assertFalse
-import kotlin.test.assertNotNull
 import kotlin.test.assertNull
 import kotlin.test.assertTrue
 
-@MiskTest
-class FakeRedisTest {
+abstract class AbstractRedisTest {
```

Comment:
Nice, abstract test classes are a great pattern to have tests written against interfaces tested against multiple implementations